ouroboros-consensus-shelley/src/Ouroboros/Consensus/Shelley/Eras.hs

```diff
@@ -90,6 +97,9 @@ type StandardMary = MaryEra StandardCrypto
 -- | The Alonzo era with standard crypto
 type StandardAlonzo = AlonzoEra StandardCrypto
 
+-- | The Babbage era with new crypto
```

Comment:
New crypto = standard crypto? We use `StandardCrypto` in `StandardAlonzo` and `StandardBabbage`.

Reply:
Needs updating, thanks!